pkgs/flox-cli-tests/default.nix

```diff
@@ -30,6 +30,8 @@
   unixtools,
   which,
   writeShellScriptBin,
+  GENERATED_DATA ? ./../../test_data/generated,
```

Comment:
note nonblocking: after running `just gen-data` I had to do a `direnv reload` to fix `path to mock data file doesn't exist: /nix/store/909d32z5vwp26x3myxyb0jivrkdg2q53-generated/resolve/hello-unfree.json`. Not a huge deal, but it's a bit of a papercut

Reply:
This is my hack for putting multiple output files into a directory named after the job rather than generating just a single JSON file. Docs could be better but that's an armadillo for now.

Reply:
It is, but that's more of a paper cut with how the test scripts work in my opinion